### DISCUSSION TITLE
May we support webview2 app in the future?

### DISCUSSION THREAD

**Discussion by 6Tom** [sandboxie-plus/Sandboxie#2287] [Various ideas and suggestions] [answered]
Dear friends, thank you very much for your hard work.

I found that all applications based on webview2 cannot run normally in the sandboxie. So may this support be added in the future?

Love you guys so much.

**Comment by okrc** [comment, marked as answer]
You can try add `ExternalManifestHack=msedgewebview2.exe,y`

**Reply by 6Tom** [reply]
Wow this is awesome. It works for many webview2 apps. Thank you very much, have a nice day.

**Reply by isaak654** [reply]
That's interesting, do you think this is mature enough to be added as part of [[Template_Edge_Fix]](https://github.com/sandboxie-plus/Sandboxie/blob/05b9ffcad5010cd4743419944a374f85846fe24e/Sandboxie/install/Templates.ini#L642) ?

**Reply by okrc** [reply]
> That's interesting, do you think this is mature enough to be added as part of [[Template_Edge_Fix]](https://github.com/sandboxie-plus/Sandboxie/blob/05b9ffcad5010cd4743419944a374f85846fe24e/Sandboxie/install/Templates.ini#L642) ?

I think it's OK. ~However msedge still doesn't open in `Security hardened Sandbox` now, I'm not sure if this problem existed before, msedgewebview2 has the same problem.~

**Reply by isaak654** [reply]
Strange, can you confirm that you're on [v1.4.2](https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.4.2)?

**Reply by okrc** [reply]
> Strange, can you confirm that you're on [v1.4.2](https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.4.2)?

Yes, this still exists in v1.4.2.

**Reply by isaak654** [reply]
See #2341

**Reply by okrc** [reply]
Yes, because I only have a Windows 22H2 environment, I did not test it on other versions of Windows.

**Reply by okrc** [reply]
Correction, it seems that most software cannot be opened. Since I just reinstalled my system, I am not currently using Sandboxie-Plus.
